src/main/java/com/quirko/gui/Main.java

```diff
@@ -14,21 +16,24 @@
 
     @Override
     public void start(Stage primaryStage) throws Exception {
-
         URL location = getClass().getClassLoader().getResource("gameLayout.fxml");
         ResourceBundle resources = null;
         FXMLLoader fxmlLoader = new FXMLLoader(location, resources);
         Parent root = fxmlLoader.load();
         GuiController c = fxmlLoader.getController();
-
+        DifficultySetPanel dfs= new DifficultySetPanel();
+        Enum diff = dfs.getDiff();
+        while(diff == null){

```

Comment:
Burada bir code smell var. Arayüz kodunda bir tasarım hatası var galiba. Burada bir döngünün olması çok garip.

"A code smell is a surface indication that usually corresponds to a deeper problem in the system."

Reply:
Hocam burada başlangıçta null olarak tanımlı olduğu için diff (difficultyLevel) butona basıp değer algılanınca çalışması açısından böyle yazdım. Yukarıda da sleeple bu yüzden  saniye bekletiyorum.